### DISCUSSION TITLE
on the wifi module side, what firmware should I use

### DISCUSSION THREAD

**Discussion by crazydiamondzgy** [khoih-prog/WiFiNINA_Generic#13] [General]
eg: if I use wifi module like ESP32-S3,  what firmware should I use, thanks in advance

**Comment by khoih-prog** [comment]
You have to use ESP32-related libraries, such as

1. Built-in [ESP32 core's **WiFi**](https://github.com/espressif/arduino-esp32/tree/master/libraries/WiFi)
2. [**ESP_WiFiManager**](https://github.com/khoih-prog/ESP_WiFiManager)
3. [**ESPAsync_WiFiManager**](https://github.com/khoih-prog/ESPAsync_WiFiManager)
4. [**ESP_WiFiManager_Lite**](https://github.com/khoih-prog/ESP_WiFiManager_Lite)
5. [**ESPAsync_WiFiManager_Lite**](https://github.com/khoih-prog/ESPAsync_WiFiManager_Lite)

**Comment by crazydiamondzgy** [comment]
thanks




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "khoih-prog/WiFiNINA_Generic"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年10月16日(星期六) 晚上11:21
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [khoih-prog/WiFiNINA_Generic] on the wifi module side, what firmware should I use (Discussion #13)





 
You have to use ESP32-related libraries, such as
  
Built-in ESP32 core's WiFi
 
ESP_WiFiManager
 
ESPAsync_WiFiManager
 
ESP_WiFiManager_Lite
 
ESPAsync_WiFiManager_Lite
  
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by JAndrassy** [comment]
@khoih-prog , I think the answer here is [nina-fw](https://github.com/arduino/nina-fw)
